src/utils/fetch-bounties.ts

```diff
@@ -18,39 +16,36 @@ export async function fetchBountiesAndChildBounties(showProgress = true) {
 		if (showProgress) {
 			showLoadingDialog('Loading...');
 		}
-		const api = get(dotApi);
-		await api.query.System.BlockWeight.getValue(); // somehow makes the next query work
-		const currentBlock = await api.query.System.Number.getValue();
-
-		const rawBounties = await api.query.Bounties.Bounties.getEntries();
-		const bounties = rawBounties
-			.map(({ value, keyArgs: [id] }) => parseBounty(value, id, currentBlock))
-			.sort((a, b) => (a.id > b.id ? -1 : 1));
 
-		const bountiesMap = keyBy(bounties, 'id');
-		const descriptions = await api.query.Bounties.BountyDescriptions.getEntries();
-		descriptions.forEach(({ value, keyArgs: [id] }) => {
-			if (id in bountiesMap) {
-				bountiesMap[id].description = value.asText();
-			}
-		});
+		const bounties = await fetchBountiesFromBlockchain();
 
-		const rawChildBounties = await api.query.ChildBounties.ChildBounties.getEntries();
-		const childBounties = rawChildBounties
-			.map(({ value, keyArgs: [, id] }) => parseChildBounty(value, id, currentBlock))
-			.sort((a, b) => (a.id > b.id ? -1 : 1));
+		try {
+			// TODO: skip for Paseo
+			// get optional data about inactive bounties from doTreasury API
+			await addBountiesFromDoTreasury(bounties);
+		} catch (exception) {
+			// log the error for developers but continue normally even if doTreasury is down
```

Comment:
should we have some UI element to indicate that inactive bounties are not shown? we could use an error dialog, but that's too much?

Reply:
When we were discussing this possibility with Vasilis he didn’t come up with an idea of a UI indication. I don’t think this has to be a part of this PR.

Reply:
Sure, I just wanted to raise it, in case it's needed.